engine/src/main/java/org/terasology/persistence/internal/StoragePathProvider.java

```diff
@@ -26,6 +26,7 @@
 public class StoragePathProvider {
     private static final String PLAYERS_PATH = "players";
     private static final String WORLDS_PATH = "worlds";
+    private static final String PREVIEWS_PATH = "previews";
```

Comment:
Rather than add a unique path for previews I suggest we just consider regular screenshots and previews the same thing and store both in a "screenshots" in the game save dir. I'm not sure if we should duplicate screenshots to a global "screenshots" in the data dir root or even have one at all (the three tile images are just for debugging and could probably be disabled or get an option to enable them somewhere)

Reply:
From my perspective, we should not mix user's screenshots and random preview images. Generally, because they are for different purposes. And it might be real pain in the neck to manage copies or links in global screenshot folder for all existing and old saved games.